src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.Tests/ServiceControllerTests.cs

```diff
@@ -87,9 +87,16 @@ private static bool RunningWithElevatedPrivileges
             get { return s_runningWithElevatedPrivileges.Value; }
         }
 
-        private void AssertExpectedProperties(ServiceController testServiceController)
+        private void AssertExpectedProperties(ServiceController testServiceController, bool serviceNameToUpper = false)
         {
-            Assert.Equal(_testService.TestServiceName, testServiceController.ServiceName);
+            if (serviceNameToUpper)
```

Comment:
Because this test method is used in pretty much all the tests inside this class, so if we do this approach all of the other tests that are creating the ServiceController with a lowercase will fail as the expected will be uppercase and the actual lowercase. So I added this parameter so that the only test that actually asserts with uppercase value is the test that is creating the ServiceController with uppercase name.